src/Service/MagazineManager.php

```diff
@@ -70,6 +72,9 @@ public function create(MagazineDto $dto, User $user, bool $rateLimit = true): Ma
         $magazine = $this->factory->createFromDto($dto, $user);
         $magazine->apId = $dto->apId;
         $magazine->apProfileId = $dto->apProfileId;
+        if ($magazine->apId) {
+            $magazine->apPublicUrl = $dto->apProfileId;
```

Comment:
> Just want to double check that this is safe to do, or should it only happen when `apPublicUrl` isn't set?
> 
> If you run `select count(*) from magazine where ap_id is not null and ap_public_url != ap_profile_id;` do you get back many magazines? Well, maybe not you, since you probably have this change which if it was different would always lead to a non-zero value (if it is zero, why do we even have the two columns in the first place, I imagine there must be a reason?)

![image](https://github.com/MbinOrg/mbin/assets/35878315/182ce397-1800-4b05-8bcf-479920c2b651)

Reply:
What I did is just an initial set of the variable, because it is needed so the update works. The update does use the apPulicUrl and the dto does not have that field.
The problem is, that magazines which do not provide the `url` property in their json response can no be updated right now because that field is null.
I guess the 2 magazines that don't have matching public url and profile id have no public url at all

Reply:
Nevermind, that is the wrong fix...

Reply:
There was only one call that used `apPublicUrl` and I fixed that to use the `apProfileId` like every other `UpdateActorMessage` dispatch uses